cmd/server/assets/login/registerphone.html

```diff
@@ -50,7 +50,9 @@ <h1>SMS auth registration</h1>
           </div>
         </form>
 
-        <a id="skip" class="card-link" href="/realm">Skip for now</a>
+        {{ if ne .mfamode 1}}
```

Comment:
This feels too magical to me. Could we add a method on the realm like `RequiresMFA` or something so this becomes:

```
{{ if $currentRealm.RequireMFA }}
```

"1" just feels magical and isn't very descriptive.

Reply:
magic numbers begone